frontend-site/src/modules/staticData.js

```diff
@@ -1,29 +1,51 @@
 // TODO: decide what stays here and what can be stored as a JSON file for the API to send
 export const utilitiesOffered = Object.freeze([
+  '3-D Printing',
   'Raspberry Pi',
   'Arduino',
+  'LUG API',
+  'Board Game Night',
   'Ice Cream',
-  '3-D Printing',
 ]);
 
 export const servicesOffered = Object.freeze([
   // TODO: fill in empty entries
+  {
+    name: '3D Printing',
+    image: '',
+    description: 'Becoming a LUG member brings many perks such as having easy access to our 3D printer located in the ACM/LUG office. Members can watch their creations come alive for minimal cost.',
+    url: 'https://lug.cs.uic.edu/3dprint/home',
+  },
   {
     name: 'Raspberry Pi',
     image: '',
-    description: '',
+    description: 'At the center of student creativity in computing is the infamous Rasberry Pi, and LUG is here to help students gain access to one. We offer Rasberry Pi rentals for any student interested.',
     url: '',
   },
   {
     name: 'Arduino',
     image: '',
-    description: '',
+    description: 'If your looking for a microcontroller but dont want to buy your own, LUG has you covered. In addition to the Rasberry Pi, LUG offers Arduino rentals.',
     url: '',
   },
   {
-    name: '3D Printing',
+    name: 'LUG API',
     image: '',
-    description: '',
-    url: 'https://lug.cs.uic.edu/3dprint/home',
+    description: 'LUG is determined to present students with tools that will help them on their road to success, which is why we made our API open to anyone, presenting students with the opportunity to see how this technology works.',
+    url: '',
+  },
+  {
+    name: 'Board Game Night',
+    image: '',
+    description: 'One Friday a month, LUG hosts a board game night. If you need a break from the stress of college and want to participate in someting fun, gamenight is the thing to do. Sometimes the game nights are also accompanied by pizza.',
+    url: '',
+  },
+  {
+    name: 'Ice Cream',
+    image: '',
+    description: 'Ice cream, being an important part of a students diet, is available for purchase in the LUG office.',
```

Comment:
Spelling/Grammar: students -> student's.

Also @Benbigmac, isn't it usually worded as "mandatory donation" and not "purchase"?

Reply:
It is worded as mandatory donation, because otherwise we'd have to deal with sales tax and the IRS and we don't mess with them Tax boys